proto/lib/protocol/bep.proto

```diff
@@ -147,6 +148,29 @@ message Counter {
     uint64 value = 2;
 }
 
+message PlatformData {
+    UnixData    unix    = 1 [(gogoproto.nullable) = true];
+    WindowsData windows = 2 [(gogoproto.nullable) = true];
+}
```

Comment:
This will grow with more entries soon, to enable xattr support.

Reply:
Should this be called ownership then/oppose to unix/windows?
Or is that going to go under *Data?

Reply:
I was planning to stuff windows xattrs in windows platform data, and invent new linux/freebsd/macOS/etc platform data fields for xattrs for those platforms. The reason being that even if the apis are similar the xattrs for one don't apply to the other. (And also add a non-wire-sent inode modtime field to detect changes.)